test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -780,6 +780,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
         $result.Output.RelationLink["next"] | Should -BeExactly "${baseUri}?maxlinks=3&linknumber=2&type=${type}"
     }
 
+    It "Verify Invoke-WebRequest supresses terminating errors with -SkipHttpErrorCheck" {
+        $Query = @{
+            statuscode = 404
+            responsephrase = 'Not found'
+            contenttype = 'text/plain'
+            body = 'oops'
+            headers = "{}"
+        }
+
+        $Uri =  Get-WebListenerUrl -Test 'Response' -Query $Query
+        $command = "Invoke-WebRequest -SkipHttpErrorCheck -Uri '$uri'"
+        $result = ExecuteWebCommand -command $command
```

Comment:
Please start parameter names with upper case
```suggestion
        $result = ExecuteWebCommand -Command $command
```
Below too.
 Should -be ->  Should -Be 
and so on.

Reply:
Done. I've also fixed some inconsistent capitalization with variable names.